GVFS/GVFS.Common/Maintenance/PrefetchStep.cs

```diff
@@ -259,8 +259,13 @@ private bool TryGetMaxGoodPrefetchTimestamp(out long maxGoodTimestamp, out strin
             return true;
         }
 
-        private bool TrySchedulePostFetchJob(List<string> packIndexes)
+        private void TrySchedulePostFetchJob(List<string> packIndexes)

```

Comment:
This `bool` to `void` change is because the only caller does not use the return type.

Reply:
Should we change the method name as well then and remove the `Try`?

Reply:
An excellent point.